README.md

```diff
@@ -180,11 +103,6 @@ After you create a development app (`bundle exec rake development_app`):
 | [Proposals](https://github.com/AjuntamentdeBarcelona/decidim/tree/master/decidim-proposals) | The Proposals module adds one of the main features of Decidim: allows users to contribute to a participatory process by creating proposals. |
 | [System](https://github.com/AjuntamentdeBarcelona/decidim/tree/master/decidim-system) | Multitenant Admin to manage multiple organizations in a single installation |
 
-## Further configuration
-
-- [Social providers integration](https://github.com/AjuntamentdeBarcelona/decidim/blob/master/docs/social_providers.md): Enable sign up from social networks.
-- [Analytics](https://github.com/AjuntamentdeBarcelona/decidim/blob/master/docs/analytics.md): How to enable analytics
-
 ## Technical tradeoffs
```

Comment:
I'd move this to the "If you're an advanced user" that I'm talking on the other comment.